src/Admin/Admin.php

```diff
@@ -43,6 +43,9 @@ public function init() {
 		$this->admin_enabled    = apply_filters( 'graphql_show_admin', true );
 		$this->graphiql_enabled = apply_filters( 'graphql_enable_graphiql', get_graphql_setting( 'graphiql_enabled', true ) );
 
+		$admin_notices = new AdminNotices();
```

Comment:
This should probably be wrapped in an `is_admin()` check. I don't think we're doing that earlier in the lifecycle but I might be wrong (reviewing via 📱)